app/src/main/java/org/wikipedia/dataclient/mwapi/TemplateDataResponse.kt

```diff
@@ -0,0 +1,53 @@
+package org.wikipedia.dataclient.mwapi
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.booleanOrNull
+import kotlinx.serialization.json.contentOrNull
+import kotlinx.serialization.json.decodeFromJsonElement
+import kotlinx.serialization.json.jsonPrimitive
+import org.wikipedia.json.JsonUtil
+
+@Serializable
+class TemplateDataResponse : MwResponse() {
```

Comment:
Would it be cleaner if this was moved into `MwQueryPage`?
The response is already an instance of Page, with parameters like `title` and `description` already there. The only new stuff is the Params class, which can be a new inner class like e.g. `Revision` or `EntityTerms` that are possible in `MwQueryPage`.